src/mdns/mdns_avahi.cpp

```diff
@@ -829,6 +869,44 @@ otbrError PublisherAvahi::UnpublishHost(const char *aName)
     return error;
 }
 
+otbrError PublisherAvahi::TxtListToAvahiStringList(const TxtList &   aTxtList,
```

Comment:
Is it related change?

Reply:
Yes. I extracted this function to let it be reused.